app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java

```diff
@@ -176,6 +181,8 @@ public Fragment getItem(int position) {
                         return InvertingOpAmpExperiment.newInstance();
                     case "Non Inverting Op-Amp":
                         return NonInvertingOpAmpExperiment.newInstance();
+                    case "RC Phase Shift":
+                        return RCPhaseShiftExperiment.newInstance();
                     case "Precision Rectifier":
```

Comment:
I didn't remove. It wasn't there in the first place. It would appear after Summing Junction experiment is merged.